03/Cart.js

```diff
@@ -1,7 +1,31 @@
+'use strict';
 import React from 'react';
+import Product from './Product';
 
-function Cart(props) {
-    return <h2>Cart</h2>
+class Cart extends React.Component {
+  render() {
+    const { cart, onAddToCart, onRemoveFromCart } = this.props;
+
+    const cartList = cart.map((product) => (
+      <Product
+        key={product.id}
+        id={product.id}
+        name={product.name}
+        price={product.price}
+        onAddToCart={onAddToCart}
+        onRemoveFromCart={onRemoveFromCart}
+        isProductInCart={true}
+        isCart
```

Comment:
Można też zdefiniować jeden `props` i przekazywać do niego albo "add" albo "remove" tj. handleClick={ remove} - wiadomo, że w "cart" będzie tylko "usuwanie". To rozwiązanie może uprościć komponent Product.